urwid/vterm.py

```diff
@@ -1376,10 +1375,10 @@ def __init__(
         """
         A terminal emulator within a widget.
 
-        ``command`` is the command to execute inside the terminal, provided as a
-        list of the command followed by its arguments.  If 'command' is None,
-        the command is the current user's shell. You can also provide a callable
-        instead of a command, which will be executed in the subprocess.
+        ``command`` is the command to execute inside the terminal,
+        provided as a list of the command followed by its arguments.
+        If 'command' is None, the command is the current user's shell.
+        You can also provide a callable instead of a command, which will be executed in the subprocess.
```

Comment:
seems like you *almost* aligned this paragraph to a new line width

Reply:
I changed split to increase readability: misread caused incomplete type annotation